lib/puppet/indirector/msgpack.rb

```diff
@@ -62,13 +62,13 @@ def load_msgpack_from_file(file, key)
     rescue Errno::ENOENT
       return nil
     rescue => detail
-      raise Puppet::Error, "Could not read MessagePack data for #{indirection.name} #{key}: #{detail}", detail.backtrace
+      raise Puppet::Error, _("Could not read MessagePack data for %{indirection} %{key}: %{detail}") % { indirection: indirection.name, key: key, detail: detail }, detail.backtrace
```

Comment:
Could you add a note that `MessagePack` refers to an application name (ie not just a pack of messages)